.spectral.yaml

```
@@ -156,6 +156,17 @@ rules:
         functionOptions:
           match: "^(dev|qa|stage|prod)(,(dev|qa|stage|prod))*$"
 
+  no-slash-before-custom-method:
+  description: "Custom methods (e.g., ':applyItem') should not be preceded by a '/'."
+  message: "The path '{{path}}' contains a '/' before a custom method. Custom methods should not start with a '/'."
+  severity: error
+  given: "$.paths"
+  then:
+    field: "@key"
+    function: pattern
+    functionOptions:
+      notMatch: "^/[^/]+/:[a-zA-Z]+$"
```

```suggestion
      notMatch: "/[^/]+/:[a-zA-Z]+$"
```
Example that would not be caught originally /api/atlas/v2/groups/{groupId}/:migrate.

 I also had to add indents to get it to run but not sure if this is just me